test/async-hooks/test-filehandle-no-reuse.mjs

```diff
@@ -0,0 +1,64 @@
+import { hasCrypto, skip, mustCall } from '../common/index.mjs';
+import initHooks from './init-hooks.mjs';
+import { checkInvocations } from './hook-checks.mjs';
+import { path } from '../common/fixtures.mjs';
+if (!hasCrypto)
+  skip('missing crypto');
+import { constants, createServer, connect } from 'http2';
```

Comment:
Here as well, named imports doesn't improve the readability IMO (`constants` on its own is a very obscure name)
```suggestion
import http2 from 'http2';
```